Sources/SwiftProtobuf/JSONScanner.swift

```diff
@@ -1250,29 +1250,42 @@ internal struct JSONScanner {
   /// Throws if field name cannot be parsed.
   /// If it encounters an unknown field name, it silently skips
```

Comment:
This comment isn't right, we do throw for unknown fields.

Reply:
Corrected the comment to explain that it silently skips if `ignoreUnknownFields` is set and throws otherwise.